src/fastoad/model_base/tests/test_flight_point.py

```diff
@@ -80,3 +80,37 @@ def test_scalarize():
     assert fp.mass == 70000.0
     assert fp.altitude == 1000.0
     assert_allclose(fp.mach, [0.7, 0.8])
+
+
+def test_descriptors():
+    FlightPoint.add_field(
+        "foo", annotation_type=float, default_value=42.0, unit="m", cumulative=False, output=True
+    )
+    # Testing redeclaration
+    FlightPoint.add_field(
+        "foo",
+        annotation_type=float,
+        default_value=42.0,
+        unit="slug/ft",
+        cumulative=True,
+        output=False,
+    )
+
+    assert FlightPoint.get_units()["time"] == "s"
+    assert FlightPoint.get_units()["foo"] == "slug/ft"
```

Comment:
This doesn't seem like the recommended way of doing it now, is it worth testing ?

Reply:
Absolutely. I had to keep `get_units()` working because it is still used at 2 locations.